.builds/archlinux.yml

```diff
@@ -18,11 +18,21 @@ sources:
 tasks:
   - setup: |
       cd wlroots
-      CC=gcc meson build-gcc -Dauto_features=enabled -Dlogind-provider=systemd
-      CC=clang meson build-clang -Dauto_features=enabled -Dlogind-provider=systemd
+      CC=gcc meson build-gcc -Dauto_features=enabled -Dlogind-provider=systemd --prefix /usr
+      CC=clang meson build-clang -Dauto_features=enabled -Dlogind-provider=systemd --prefix /usr
   - gcc: |
       cd wlroots/build-gcc
       ninja
+      sudo ninja install
+      cd ../tinywl
+      make
+      cd ../build-gcc
+      sudo ninja uninstall
   - clang: |
       cd wlroots/build-clang
       ninja
+      sudo ninja install
+      cd ../tinywl
+      make
```

Comment:
Here tinywl is built twice with GCC, because `CC` is only set for the `meson` commands. Also I think the wlroots installation won't change a lot between GCC and Clang.

We could only build tinywl with GCC I think, because Clang is already used on FreeBSD.

Reply:
Oh right, tinywl is not built with meson. I removed the tinywl build from it and squashed the fix into the ci commit.